docs/manifest.json

```
@@ -519,6 +519,15 @@
                         }
                       ]
                     ]
+                  },
+                  {
+                    "title": "Web3",
+                    "items": [
+                      {
+                        "title": "Coinbase Wallet",
+                        "href": "/docs/authentication/web3/coinbase-wallet"
+                      }
+                    ]
```

An additional array is required here to conform to the [JSON schema](https://github.com/clerk/clerk-docs/blob/8c304af5180ac25f29a880167eca36bb71f618fc/docs/manifest.schema.json):

```suggestion
                    "items": [
                      [
                        {
                          "title": "Coinbase Wallet",
                          "href": "/docs/authentication/web3/coinbase-wallet"
                        }
                      ]
                    ]
```

---

Thanks!